client/src/components/create-page/CreateProfile.css

```
@@ -1,95 +1,127 @@
-.create-profile-page {
+.background-create-page {
   background-color: var(--light-blue);
```

```suggestion
  background-color: var(--background-light);
```

"It’s better for the (variable) to reflect what it is than what it looks like."

https://css-tricks.com/what-do-you-name-color-variables/